testnet/stacks-node/src/config.rs

```diff
@@ -2408,6 +2409,10 @@ impl EventKeyType {
             return Some(EventKeyType::BlockProposal);
         }
 
+        if raw_key == "stacker_set" {
```

Comment:
I see we have the means to subscribe to stacker sets, but this doesn't help a signer that spins up after stackers have registered. Should I create a ticket to add this as an rpc endpoint as we will need to configure the signers with it?

Reply:
Yes, please open a ticket -- it should be simple enough to add in a follow-up PR.

Reply:
Done https://github.com/stacks-network/stacks-core/issues/4298